src/backend/_scss/application.scss

```diff
@@ -1251,3 +1251,97 @@ a {
   right :0;
   top :0;
 }
+
+.c-tabs-nav {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.c-tabs-nav__link {
+  flex: 1;
+  margin-right: 4px;
+  padding: 12px;
+  color: #fff;
```

Comment:
## Required
Use SASS nesting properties instead of using same class names .c-tabs-nav__link